nengo/probe.py

```diff
@@ -28,9 +30,10 @@ def validate(self, probe, attr):
                                 "into 'weights'. Probe 'weights' instead.",
                                 since="v2.1.0")
         if attr not in probe.obj.probeable:
-            raise ValidationError("Attribute %r is not probeable on %s.\n"
+            raise ValidationError("Attribute %r is not probeable on %s.%s"
```

Comment:
I just verified this in Python 2 and 3 in a Windows command prompt; [`sys.stdout`](https://docs.python.org/2/library/sys.html#sys.stdout) is indeed a file object which is likely where the line endings get handled... in any case, they do appear to get handled so I'll revert :rainbow: